test/specs/commands/createTransactionCastVotes.js

```diff
@@ -0,0 +1,259 @@
+											Then('it should not get the inputs from sources using the second passphrase source', then.itShouldNotGetTheInputsFromSourcesUsingTheSecondPassphraseSource);
+											Then('it should not get the data using the votes source', then.itShouldNotGetTheDataUsingTheVotesSource);
+											Then('it should not get the data using the unvotes source', then.itShouldNotGetTheDataUsingTheUnvotesSource);
+											Then('it should create a cast vote transaction with the passphrase and the public keys prepended with the correct modifier', then.itShouldCreateACastVoteTransactionWithThePassphraseAndThePublicKeysPrependedWithTheCorrectModifier);
+											Then('it should resolve to the created transaction', then.itShouldResolveToTheCreatedTransaction);
+										});
+									});
+									Given('an options object with votes set to "file:path/to/votes.txt" and unvotes set to "file:path/to/unvotes.txt"', given.anOptionsObjectWithVotesSetToAndUnvotesSetTo, () => {
+										Given('the vote can be retrieved from its source', given.theVotesCanBeRetrievedFromItsSource, () => {
+											Given('the unvotes can be retrieved from its source', given.theUnvotesCanBeRetrievedFromItsSource, () => {
+												When('the action is called with the options', when.theActionIsCalledWithTheOptions, () => {
+													Then('it should get the inputs from sources using the Vorpal instance', then.itShouldGetTheInputsFromSourcesUsingTheVorpalInstance);
+													Then('it should get the inputs from sources using the passphrase source with a repeating prompt', then.itShouldGetTheInputsFromSourcesUsingThePassphraseSourceWithARepeatingPrompt);
+													Then('it should not get the inputs from sources using the second passphrase source', then.itShouldNotGetTheInputsFromSourcesUsingTheSecondPassphraseSource);
+													Then('it should get the data using the votes source', then.itShouldGetTheDataUsingTheVotesSource);
+													Then('it should get the data using the unvotes source', then.itShouldGetTheDataUsingTheUnvotesSource);
+													Then('it should create a cast vote transaction with the passphrase and the public keys prepended with the correct modifier', then.itShouldCreateACastVoteTransactionWithThePassphraseAndThePublicKeysPrependedWithTheCorrectModifier);
+													Then('it should resolve to the created transaction', then.itShouldResolveToTheCreatedTransaction);
+												});
+											});
+										});
+									});
+								});
+								Given('an options object with votes set to "file:path/to/votes.txt" and passphrase set to "prompt"', given.anOptionsObjectWithVotesSetToAndPassphraseSetTo, () => {
+									Given('the passphrase can be retrieved from its source', given.thePassphraseCanBeRetrievedFromItsSource, () => {
+										Given('the vote can be retrieved from its source', given.theVotesCanBeRetrievedFromItsSource, () => {
```

Comment:
We should use plural `votes` consistently (and therefore `their` rather than `its` - also in the function name).